testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -275,6 +275,13 @@ def params_from_base_suite_setup(request):
             testserver.install_device()
         else:
             testserver.install()
+        test_name_cp = test_name.replace("/", "-")
```

Comment:
is the testserver.start a duplicated statement? it starts some lines below after server/sgw provisioning code.

Reply:
removed